### PR TITLE
work around of the build break in mac

### PR DESCRIPTION
**Pull request by yufenglee**
Workaround to fix a build break in mac. Please refer to issue: https://github.com/actions/virtual-environments/issues/1811

### CODE REVIEW

**Review by gwang-msft — DISMISSED** [review]

**Comment by snnn** [comment]
There are 5 places.

1. github/azure-pipelines/templates/mac-ci.yml
2. github/azure-pipelines/templates/py-packaging-stage.yml
3. github/azure-pipelines/c-api-packaging-pipelines.yml
4. github/azure-pipelines/android-x86_64-crosscompile-ci-pipeline.yml
5. github/azure-pipelines/nuget/templates/test_macos.yml

File: tools/ci_build/github/azure-pipelines/templates/mac-ci.yml [hunk 1]
```diff
@@ -39,6 +39,8 @@ jobs:
         export CMAKE_ARGS="-DONNX_GEN_PB_TYPE_STUBS=OFF -DONNX_WERROR=OFF"
         sudo python3 -m pip install -r '$(Build.SourcesDirectory)/tools/ci_build/github/linux/docker/scripts/requirements.txt'
         sudo xcode-select --switch /Applications/Xcode_10.app/Contents/Developer
+        # The following line is a hack to prevent build pipeline from failing
+        brew uninstall openssl@1.0.2t
```
Comment (snnn) [inline tools/ci_build/github/azure-pipelines/templates/mac-ci.yml:43]: What if the version doesn't exist?

**Comment by pranavsharma** [comment]
> There are 5 places.
> 
> 1. github/azure-pipelines/templates/mac-ci.yml
> 2. github/azure-pipelines/templates/py-packaging-stage.yml
> 3. github/azure-pipelines/c-api-packaging-pipelines.yml
> 4. github/azure-pipelines/android-x86_64-crosscompile-ci-pipeline.yml
> 5. github/azure-pipelines/nuget/templates/test_macos.yml

Also tools\ci_build\github\azure-pipelines\java-api-packaging-pipelines.yml

File: tools/ci_build/github/azure-pipelines/templates/mac-ci.yml (hunk 1 above)
Comment (gwang-msft) [reply tools/ci_build/github/azure-pipelines/templates/mac-ci.yml:43]: You probably need a shell script
```
if brew list --version openssl | grep '1.0.2t'; then
  brew uninstall openssl@1.0.2t
fi
```

**Comment by gwang-msft** [comment]
> There are 5 places.
> 
> 1. github/azure-pipelines/templates/mac-ci.yml
> 2. github/azure-pipelines/templates/py-packaging-stage.yml
> 3. github/azure-pipelines/c-api-packaging-pipelines.yml
> 4. github/azure-pipelines/android-x86_64-crosscompile-ci-pipeline.yml
> 5. github/azure-pipelines/nuget/templates/test_macos.yml

Android CI is running on MacOS 10.15, current don't need this work around

**Comment by snnn** [comment]
> Android CI is running on MacOS 10.15, currently it doesn't need this work around

Or can we just make them consistent, make all of them use 10.15?

**Comment by yufenglee** [comment]
> > Android CI is running on MacOS 10.15, currently it doesn't need this work around
> 
> Or can we just make them consistent, make all of them use 10.15?

good finding. let me make a PR to use 10.15 and see if it can pass.

**Comment by yufenglee** [comment]
> > > Android CI is running on MacOS 10.15, currently it doesn't need this work around
> > 
> > 
> > Or can we just make them consistent, make all of them use 10.15?
> 
> good finding. let me make a PR to use 10.15 and see if it can pass.

The updating fails. For urgency, make this workaround for now. Do the real fix later.

**Comment by snnn** [comment]
"brew uninstall openssl@1.0.2t brew install coreutils ninja"

That's was because you put two commands in one line.

**Review by snnn — APPROVED** [review]

**Review by gwang-msft — APPROVED** [review]
Do you want to be safer to check if openssl@1.0.2t is installed before uninstall?

**Pull request merged**